### ISSUE TITLE
Conflicting version in Edgeware.SR2

### ISSUE THREAD

**Issue by jo2** [spring-cloud/spring-cloud-netflix#3119]
I'm using `spring-boot-starter-data-cassandra` in the `1.5.10-RELEASE` as the rest of my spring boot dependencies. In addition to that I'm using `spring-cloud-starter-netflix-eureka-client` in the `Edgeware.SR2` as recommended [here](https://spring.io/blog/2018/02/09/spring-cloud-edgware-sr2-released). Both versions should be compatible but `spring-cloud-starter-netflix-eureka-client` uses `netty` in the `4.0.27.Final` and `spring-boot-starter-data-cassandra` uses `netty` in the `4.0.37.Final` which results in conflicts.

I hope this is the correct place to report this.

**Comment by ryanjbaxter** [comment]
The netty dependency in `spring-cloud-starter-netflix-eureka-client` is brought in via the Ribbon dependency.  I am not sure what we can really do about the conflict though.

**Comment by jo2** [comment]
In my case I added exclusions for `netty-codec-http` and `netty-transport-native-epoll` to the `spring-cloud-starter-netflix-eureka-client`.

```
<dependency>
    <groupId>org.springframework.cloud</groupId>
    <artifactId>spring-cloud-starter-netflix-eureka-client</artifactId>
    <exclusions>
        <exclusion>
            <groupId>io.netty</groupId>
            <artifactId>netty-codec-http</artifactId>
        </exclusion>
        <exclusion>
            <groupId>io.netty</groupId>
            <artifactId>netty-transport-native-epoll</artifactId>
        </exclusion>
    </exclusions>
</dependency>
```

But I'm not really happy with this soultion because in my oppinion it should not happen in copatible versions. Is it possible to add such exclusions in one of spring-cloud-netflix laysers and just add the compatible version?

**Comment by spencergibb** [comment]
I'm pretty sure we don't use anything in ribbon that uses netty. My guess is we could add those exclusions and not have any problems.

**Comment by ryanjbaxter** [comment]
@spencergibb I suppose we can get rid of these too 
https://github.com/spring-cloud/spring-cloud-netflix/blob/1.4.x/pom.xml#L123

**Comment by ryanjbaxter** [comment]
Closed via f19c7fb
